my-profile-webapp/src/main/webapp/my-app/lec/services.js

```diff
@@ -41,6 +41,24 @@ define(['angular'], function(angular) {
              });
         };
         
+        var getEmergencyPhoneNumber = function(){
+            return $http.get('/profile/api/emergencyPhoneNumber/get.json').success(
+                function(data, status){
+                    return data;
+                }).error(function(data, status){
+                    miscService.redirectUser(status, "Get emergency contact info");
```

Comment:
I would recommend not making it default as it would be confusing to have a log warning that said "misc service error"